### DISCUSSION TITLE
Is it possible to create multiple executables using the cxfreeze script?

### DISCUSSION THREAD

**Discussion by ubalklen** [marcelotduarte/cx_Freeze#2199] [Q&A] [answered]
I want to do what has been proposed in this [SO answer](https://stackoverflow.com/a/42438235/17987266):

```python
from cx_Freeze import setup, Executable

setup(
        name = "hello",
        version = "0.1",
        description = "the typical 'Hello, world!' script",
        executables = [Executable("hello.py"), Executable("hello1.py"), Executable("hello2.py")]
)
```
But using the cxfreeze's CLI. Is this possible?

**Comment by marcelotduarte** [comment, marked as answer]
You can get multiple executables only using setup script.
I'm working on a version that you get multiple executables using cxfreeze using a pyproject.toml
